recipe/setupext.patch

```diff
@@ -1,16 +1,15 @@
-diff --git setupext.py setupext.py
-index 0ae553a..66c6e02 100755
---- setupext.py
-+++ setupext.py
-@@ -1425,6 +1425,7 @@ class BackendTkAgg(OptionalBackendPackage):
-             tk.destroy()
- 
-         # Save directories and version string to cache
-+#       tcl_lib_dir = tk_lib_dir = r'C:\Tcl%d' % (8 * tuple.__itemsize__)
-         self.tcl_tk_cache = tcl_lib_dir, tk_lib_dir, str(Tkinter.TkVersion)[:3]
-         return self.tcl_tk_cache
- 
-@@ -1531,13 +1532,10 @@ class BackendTkAgg(OptionalBackendPackage):
+--- setupext.py	2016-01-10 19:20:20.000000000 -0300
++++ setupext.py	2016-05-14 16:29:26.477374144 -0300
+@@ -990,7 +990,7 @@
+             ]
+         ext = make_extension('matplotlib._png', sources)
+         pkg_config.setup_extension(
+-            ext, 'libpng', default_libraries=['png', 'z'],
++            ext, 'libpng', default_libraries=['libpng', 'z'],
```

Comment:
I am usually clueless on Windows, but the [`libpng`](http://anaconda.org/conda-forge/libpng/1.6.21/download/win-64/libpng-1.6.21-vc14_1.tar.bz2) version we have for Windows at `conda-forge` has a `libpng.lib` file. However, without the patch, matplotlib cannot find it. Not sure what is going on there...